### DISCUSSION TITLE
请问为啥设置完 AKISMET_KEY 没有效果？

### DISCUSSION THREAD

**Discussion by ethan-enhe** [walinejs/waline#894] [Q&A] [answered]
rt，似乎发个全是数字的垃圾评论也没有被标记为垃圾（

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/blob/main/packages/server/src/service/akismet.js#L7 Waline 默认有一个 Akismet Key，所以这个功能是默认开启的，不需要设置。  
https://github.com/walinejs/waline/blob/main/packages/server/src/controller/comment.js#L388 Waline 有在后台日志中输出 Akismet 检测结果，你可以在日志中查看检测状态。具体检测规则我们不做处理。
如果你觉得 Akismet 的检测规则比较弱，你也可以通过 `preSave` 钩子自定义检测规则，https://waline.js.org/reference/server.html#presave-comment。

**Reply by ethan-enhe** [reply]
感谢
